app/src/main/java/org/mozilla/fenix/share/ShareController.kt

```diff
@@ -172,7 +172,7 @@ class DefaultShareController(
 
     @VisibleForTesting
     fun getShareText() = shareData.joinToString("\n\n") { data ->
-        listOfNotNull(data.title, data.url).joinToString("\n")
+        data.url ?: ""
```

Comment:
tip: there's a helper for this!
```suggestion
        data.url.orEmpty()
```

Reply:
👀 didn't know that. thanks!